src/workloads/query/multiplanner/MultiplannerWithGroup.yml

```diff
@@ -0,0 +1,399 @@
+      - keys: {x54: 1}
+        options: {name: index54}
+      - keys: {x55: 1}
+        options: {name: index55}
+      - keys: {x56: 1}
+        options: {name: index56}
+      - keys: {x57: 1}
+        options: {name: index57}
+      - keys: {x58: 1}
+        options: {name: index58}
+      - keys: {x59: 1}
+        options: {name: index59}
+      - keys: {x60: 1}
+        options: {name: index60}
+      - keys: {x61: 1}
+        options: {name: index61}
+      - keys: {x62: 1}
+        options: {name: index62}
+      - keys: {x63: 1}
+        options: {name: index63}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: MultiplannerWithGroup64CompetingIndexes
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: AggregationTemplate
+    Parameters:
+      database: *db
+      active: [3]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      collection: *coll
+      pipeline: &pipeline [{$match: {
+            x1: {$lte: 1},
+            x2: {$lte: 1},
+            x3: {$lte: 1},
+            x4: {$lte: 1},
+            x5: {$lte: 1},
+            x6: {$lte: 1},
+            x7: {$lte: 1},
+            x8: {$lte: 1},
+            x9: {$lte: 1},
+            x10: {$lte: 1},
+            x11: {$lte: 1},
+            x12: {$lte: 1},
+            x13: {$lte: 1},
+            x14: {$lte: 1},
+            x15: {$lte: 1},
+            x16: {$lte: 1},
+            x17: {$lte: 1},
+            x18: {$lte: 1},
+            x19: {$lte: 1},
+            x20: {$lte: 1},
+            x21: {$lte: 1},
+            x22: {$lte: 1},
+            x23: {$lte: 1},
+            x24: {$lte: 1},
+            x25: {$lte: 1},
+            x26: {$lte: 1},
+            x27: {$lte: 1},
+            x28: {$lte: 1},
+            x29: {$lte: 1},
+            x30: {$lte: 1},
+            x31: {$lte: 1},
+            x32: {$lte: 1},
+            x33: {$lte: 1},
+            x34: {$lte: 1},
+            x35: {$lte: 1},
+            x36: {$lte: 1},
+            x37: {$lte: 1},
+            x38: {$lte: 1},
+            x39: {$lte: 1},
+            x40: {$lte: 1},
+            x41: {$lte: 1},
+            x42: {$lte: 1},
+            x43: {$lte: 1},
+            x44: {$lte: 1},
+            x45: {$lte: 1},
+            x46: {$lte: 1},
+            x47: {$lte: 1},
+            x48: {$lte: 1},
+            x49: {$lte: 1},
+            x50: {$lte: 1},
+            x51: {$lte: 1},
+            x52: {$lte: 1},
+            x53: {$lte: 1},
+            x54: {$lte: 1},
+            x55: {$lte: 1},
+            x56: {$lte: 1},
+            x57: {$lte: 1},
+            x58: {$lte: 1},
+            x59: {$lte: 1},
+            x60: {$lte: 1},
+            x61: {$lte: 1},
+            x62: {$lte: 1},
+            x63: {$lte: 1}}}, {$group: {_id: 1, n: {$sum: '$z'}}}]
```

Comment:
[question] Why are we using `$z` here? Does it matter that `z` does not exist anywhere in the collection?